src/main.py

```diff
@@ -1,19 +1,17 @@
 from src.app import create_app
-from src.env_utils import MongoDBSettings
 from src.mongodb.mongo_setup import connect_to_mongodb, disconnect_from_mongodb
 from utils.custom_library_errors import *
 import uvicorn
 
 
 def main():
     try:
-        settings = MongoDBSettings()
-        connect_to_mongodb(settings)
+        connect_to_mongodb()
         app = create_app(library_name="AmarLibrary")
         uvicorn.run(app, host="127.0.0.1", port=8000)
```

Comment:
You should be getting this from env variables

Reply:
i got the MongoDBSettings from env